example/upload.py

```diff
@@ -21,7 +21,7 @@
 # Create a variable with a hard coded path to your file system
 file_name = '<full path to a video on the filesystem>'
 
-print 'Uploading: %s' % file_name
+print('Uploading: %s' % file_name)
```

Comment:
This example wouldn't run in Python 3, because print took on the signature of a function`print 'lkjds` -> `print('blah')`. This change will function for both 2 && 3.